test/util/test_case.hpp

```diff
@@ -121,6 +121,13 @@ namespace ngraph
                 auto value = read_binary_file<T>(filepath);
                 add_expected_output(expected_shape, value);
             }
+
+            NgraphTestCase& dump_results(bool dump = true)
+            {
+                m_dump_results = dump;
+                return *this;
```

Comment:
In order to keep the API consistent, please add `return *this;` to `set_tolerance` as well.